sklearn/tests/test_multioutput.py

```diff
@@ -102,6 +125,33 @@ def test_multi_target_sample_weights():
 n_samples, n_features = X.shape
 n_outputs = y.shape[1]
 n_classes = len(np.unique(y1))
+classes = np.column_stack(map(np.unique, (y1, y2, y3)))
+
+
+def test_multi_output_classification_partial_fit():
+    # test if multi_target initializes correctly with base estimator and fit
+    # assert predictions work as expected for predict, prodict_proba and score
+
+    sgd_linear_clf = SGDClassifier(loss='log', random_state=1)
+    multi_target_linear = MultiOutputClassifier(sgd_linear_clf)
+
+    # train the multi_target_linear and also get the predictions.
+    half_index = int(X.shape[0] / 2)
+    multi_target_linear.partial_fit(
+        X[:half_index], y[:half_index], classes=classes)
```

Comment:
should also check check:
* sample weight
* passing `classes=None` raises appropriate error